fw/http_limits.c

```diff
@@ -350,8 +348,6 @@ tfw_classify_conn_close(struct sock *sk)
 {
 	FrangAcc *ra = frang_acc_from_sk(sk);
 
-	BUG_ON(!ra);
```

Comment:
You should return here if ra == NULL. If cli = tfw_client_obtain fails. ra will be null here and we should immediately return here! otherwise we catch bug

Reply:
> You should return here if ra == NULL. If cli = tfw_client_obtain fails. ra will be null here and we should immediately return here! otherwise we catch bug

Exactly. I meant this in my comment to kernel patch.